src/components/Book.spec.js

```diff
@@ -0,0 +1,111 @@
+import { render, screen } from '@testing-library/react'
+import Book from './Book'
+import placeholder from '../images/placeholder.png'
+
+describe('Book', () => {
+  const books = [
+    {
+      id: '1',
+      bookCover: placeholder,
+      title: 'Harry Potter and the Goblet of Fire',
+      authors: ['J. K. Rowling'],
+      finished: false,
+      readingSince: '05/2021',
+      finishedSince: '',
+    },
+    {
+      id: '2',
+      bookCover: placeholder,
+      title: 'Cathedral of the Sea',
+      authors: ['Ildefonso Falcones de Sierra'],
+      finished: true,
+      readingSince: '05/2021',
+      finishedSince: '08/2021',
+    },
+  ]
+
+  it('renders book currently read', () => {
+    render(
+      <Book
+        key={books[0].id}
+        title={books[0].title}
+        authors={books[0].authors}
+        readingStatus={books[0].finished}
+        readingStatusDate={books[0].readingSince}
+        finishedSince={books[0].finishedSince}
+        bookCover={books[0].bookCover}
```

Comment:
This is a bit hard to read. I can understand why adding mock data is nicer for you, but for an external developer it would be nicer to see the props here directly. Right now I have to scroll up and down and I don't really know what you are testing. So maybe add the data here?

Reply:
You are right and I have now inserted the mock data directly into the tests and removed the const books.